Jellyfin.Api/Auth/SyncPlayAccessPolicy/SyncPlayAccessHandler.cs

```diff
@@ -42,10 +47,52 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             var userId = ClaimHelpers.GetUserId(context.User);
             var user = _userManager.GetUserById(userId!.Value);
 
-            if ((requirement.RequiredAccess.HasValue && user.SyncPlayAccess == requirement.RequiredAccess)
-                || user.SyncPlayAccess == SyncPlayAccess.CreateAndJoinGroups)
+            if (requirement.RequiredAccess == SyncPlayAccessRequirementType.HasAccess)
```

Comment:
I don't know if it's too late to change it, but if the SyncPlayUserAccessType has the `[Flag]` attribute, then you could use bit masking and avoid the `SyncPlayAccessRequirementType` enum.

Reply:
Bit masking does indeed sound more appropriate for this, I'll have to look into it.